### PR TITLE
Release v7.11.0

### PR DESCRIPTION
**Pull request by lildude**
Finally, another release of Linguist. 🎉 

- [x] Deployed to GitHub.com
- [ ] Grammars deployed Syntax highlighting service
- [ ] Deployed to Lightshow

### CODE REVIEW

File: grammars.yml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-https://bitbucket.org/Clams/sublimesystemverilog/get/default.tar.gz:
-- source.systemverilog
-- source.ucfconstraints
```
Comment (lildude) [inline grammars.yml:3]: Repo has gone AWOL. PR coming to replace this.

File: .gitmodules [hunk 2]
```diff
@@ -1010,6 +1010,7 @@
 [submodule "vendor/grammars/vscode-codeql"]
 	path = vendor/grammars/vscode-codeql
 	url = https://github.com/github/vscode-codeql
+	branch = main
```
Comment (lildude) [inline .gitmodules:1013]: This repo has done the `master` -> `main` rename dance.

File: grammars.yml (hunk 1 above)
Comment (lildude) [reply grammars.yml:3]: https://github.com/github/linguist/pull/4994

**Comment by Epokhe** [comment]
Hey @lildude, I know it's a draft but can't help but ask, I couldn't see Avro IDL support that was added with #4953, does this mean it's going to wait for the next release?

File: .gitmodules (hunk 2 above)
Comment (Alhadis) [reply .gitmodules:1013]: It's funny, I always thought `master` was a reference to the "master" disc of an audio recording (since branching off master implies a copy, similar to how phonographic discs are pressed).

Would have stayed that way, too, had people not drawn attention to it...

**Comment by lildude** [comment]
> Hey @lildude, I know it's a draft, I can't help but ask, I couldn't see Avro IDL support that was added with #4953, does this mean it's going to wait for the next release?

@Epokhe No. It's definitely in this release (the SHA in the link == the current HEAD of the `release-v7.11.0` branch):

https://github.com/github/linguist/blob/8d585c39baeac18e52089fb454e3949d68723e37/lib/linguist/languages.yml#L408-L414

You've not stated how you've determined it's not included, but I suspect it's because you can't see its grammar in the list of submodules mentioned in this PR. If that's the case, this is expected. It would indicate the grammar hasn't been updated since it was added or last updated, in this case, it hasn't changed in the last 12 days. Looking at the repo we can see it hasn't changed in over 2 years, so there is nothing to update since the grammar was added.

In short, any PR merged before I branched is included.

**Comment by lildude** [comment]
Heads up: this release is going to be delayed a little as we have a failure that isn't picked up by CI under Actions but is picked up when I test locally on macOS and also occurs in GitHub staging deploys:

This is the failure in one of the tests and is the same stack seen in staging:

```
  1) Error:
TestFileBlob#test_language:
NoMethodError: undefined method `[]' for nil:NilClass
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:133:in `token_probability'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:122:in `block in tokens_probability'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:121:in `each'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:121:in `inject'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:121:in `tokens_probability'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:107:in `block in classify'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:106:in `each'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:106:in `classify'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:80:in `classify'
    /Users/lildude/github/linguist/lib/linguist/classifier.rb:22:in `call'
    /Users/lildude/github/linguist/lib/linguist.rb:32:in `block (3 levels) in detect'
    /Users/lildude/github/linguist/lib/linguist.rb:100:in `instrument'
    /Users/lildude/github/linguist/lib/linguist.rb:31:in `block (2 levels) in detect'
    /Users/lildude/github/linguist/lib/linguist.rb:29:in `each'
    /Users/lildude/github/linguist/lib/linguist.rb:29:in `block in detect'
    /Users/lildude/github/linguist/lib/linguist.rb:100:in `instrument'
    /Users/lildude/github/linguist/lib/linguist.rb:24:in `detect'
    /Users/lildude/github/linguist/lib/linguist/blob_helper.rb:368:in `language'
    /Users/lildude/github/linguist/test/test_file_blob.rb:646:in `block in test_language'
    /Users/lildude/github/linguist/lib/linguist/samples.rb:58:in `block (2 levels) in each'
    /Users/lildude/github/linguist/lib/linguist/samples.rb:41:in `each'
    /Users/lildude/github/linguist/lib/linguist/samples.rb:41:in `block in each'
    /Users/lildude/github/linguist/lib/linguist/samples.rb:37:in `each'
    /Users/lildude/github/linguist/lib/linguist/samples.rb:37:in `each'
    /Users/lildude/github/linguist/test/test_file_blob.rb:644:in `test_language'
```

I saw this when testing before building the gem and thought this was specific to my local env as CI passed, but clearly not.

I'm looking into this now.

**Comment by lildude** [comment]
Repro'd locally on the v7.10.0 tag which I know for sure didn't have any errors so I don't think this is caused by a change we made.

**Comment by lildude** [comment]
> Repro'd locally on the v7.10.0 tag which I know for sure didn't have any errors so I don't think this is caused by a change we made.

Confirmed. This was something local to my env. A clean clone doesn't experience the issue.

**Comment by lildude** [comment]
Deployed to GitHub.com and looking good so I'm going to merge this now and tag. 

Some of the grammars won't show correct highlighting or any for new languages until the deploy to the syntax highlighting service which may take a day or to to happen as it's a holiday in the 🇺🇸 today. They will once the checkbox in the OP has been checked off by me.
